packages/css/src/components/shopping-cart/shopping-cart.template.ts

```diff
@@ -0,0 +1,139 @@
+import { ShoppingCart } from '@dso-toolkit/sources';
+
+import { html, nothing } from 'lit-html';
+
+import { buttonTemplate } from '../button/button.template';
+import { iconTemplate } from '../icon/icon.template';
+
+function accumulateItems(items: any) {
```

Comment:
Kan je deze drie functies niet typeren met `ShoppingCartItem` en/of `ShoppingCartSubitem` ?